src/App.js

```diff
@@ -1,24 +1,11 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Board from "./todo/container/Board";
+import "./App.css";
```

Comment:
Is App.css file necessary at this point?

Reply:
I'll remove this..